pymc3/distributions/continuous.py

```diff
@@ -910,14 +935,14 @@ def __init__(self, distribution, lower=-inf, upper=inf):
     def __call__(self, *args, **kwargs):
         first, args = args[0], args[1:]
 
-        return Bounded(first, self.distribution, self.lower, self.upper, *args, **kwargs)
+        return Bounded(first, self.distribution, self.lower, self.upper,
+                       *args, **kwargs)
 
     def dist(self, *args, **kwargs):
-        return Bounded.dist(self.distribution, self.lower, self.upper, *args, **kwargs)
+        return Bounded.dist(self.distribution, self.lower, self.upper,
+                            *args, **kwargs)
 
 
-Tpos = Bound(T, 0)
```

Comment:
We'll probably want to keep that for convenience, just rename it to `StudentTpos`.